dotCMS/src/main/java/com/dotmarketing/util/Config.java

```diff
@@ -300,17 +300,13 @@ private static void _refreshProperties () {
 
 
 	private final static String ENV_PREFIX="DOT_";
-	
-    private static void readEnvironmentVariables() {
-        
-        
-        System.getenv().entrySet().stream().filter(e->e.getKey().startsWith(ENV_PREFIX)).forEach(e->
-            props.addProperty(e.getKey(), e.getValue())
-        );
-        
 
-
-    }
+	private static void readEnvironmentVariables() {
+		synchronized (Config.class) {
```

Comment:
why the `synchronized` ?

Reply:
The other read methods above this one are synchronized (The ones that read from the properties files). I was able to corroborate that the properties are loaded repeated times during the start-up.  from static context and from the MainServlet initialize. I think there's no harm in making that piece synchronized especially when this can be called from different contexts.